src/main/java/com/stetsko/dao/impl/membership/Membership.java

```diff
@@ -0,0 +1,15 @@
+package com.stetsko.dao.impl.membership;
+
+import com.stetsko.dao.impl.ORMAbstractDAO;
+import com.stetsko.dao.membership.MembershipDAO;
+import com.stetsko.entity.membership.MembershipEntity;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class Membership extends ORMAbstractDAO<MembershipEntity> implements MembershipDAO {
```

Comment:
Class name should describe that this is DAO implementation, like MembershipOrmDAO